plugins/UsersManager/API.php

```diff
@@ -599,6 +752,20 @@ public function getUsersHavingSuperUserAccess()
         return $users;
     }
 
+    private function enrichUsersWithLastSeen($users)
+    {
+        $formatter = new Formatter();
+
+        $lastSeenTimes = LastSeenTimeLogger::getLastSeenTimesForAllUsers();
```

Comment:
Off topic: at some point be good to move this into the user table instead of option...